bundles/org.eclipse.jface/.settings/.api_filters

```diff
@@ -42,4 +42,12 @@
             </message_arguments>
         </filter>
     </resource>
+    <resource path="src/org/eclipse/jface/viewers/AbstractTreeViewer.java" type="org.eclipse.jface.viewers.AbstractTreeViewer">
+        <filter id="336658481">
+            <message_arguments>
+                <message_argument value="org.eclipse.jface.viewers.AbstractTreeViewer"/>
+                <message_argument value="NO_EXPAND"/>
+            </message_arguments>
+        </filter>
+    </resource>
```

Comment:
We need minor version bump for the jface plugin because of the API extension. From my understanding, this API filter should be obsolete when adding that version bump.

Reply:
Fixed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/4f7cd31362a686529d68f8ef6d0c33e2c779403a